src/daemon/command_line_args.h

```diff
@@ -46,6 +46,11 @@ namespace daemon_args
   , "Specify log file"
   , ""
   };
+  const command_line::arg_descriptor<std::size_t> arg_max_log_file_size = {
+    "max-log-file-size"
+  , "Specify maximum log file size [B]"
```

Comment:
Can we make the unit megabytes [MB]?

Reply:
Note that the generated files with be a bit over the limit (at next end of line), and that the default is about 8 kB below 100 MB, to ensure it fits within 100 MB.
But having an optional kB or MB suffix would be nice. Not sure if really worth the extra coding time though, so only if you enjoy doing this :)

Reply:
@NanoAkron I would prefer it like this, because If I use MB as a base unit I’d have to add validation that the bytes are not > MAX.